src/uwtools/tests/config/test_validator.py

```diff
@@ -126,6 +128,21 @@ def test_validate_yaml_pass(config, config_file, schema, schema_file):
     assert validator.validate_yaml(schema_file=schema_file, config_file=config_file)
 
 
+def test_validate_workflow_tags_pass():
+    # Test a fully valid config file passes schema
+    with resources.as_file(resources.files("uwtools.resources")) as resc:
+        schema_file = resc / "rocoto.jsonschema"
+    config_file = support.fixture_path("hello_workflow_tags.yaml")
+    assert validator.validate_yaml(
+        schema_file=schema_file, config_file=config_file, check_path=False
+    )
+
+    config_file = support.fixture_path("hello_workflow_invalid.yaml")
+    assert not validator.validate_yaml(
+        schema_file=schema_file, config_file=config_file, check_path=False
+    )
+
```

Comment:
Everything passes here for the tests. Before we close this ticket out, I wanted to have a discussion on if we think we should separate these two sections out into separate unit tests?

One uses a valid yaml confirming that it validates with the schema, and the other confirms that an invalid yaml fails the schema validation. I would consider separating them out since they are testing different configs. However, thinking about clean coding I'm not sure if it's useful to repeat getting the path to the schema twice. 

Perfectly fine if we feel like leaving this alone as well.

Reply:
Actually looking at this again, I tried to perform `parameterization` on this test:

```
@pytest.mark.parametrize("fn", ["hello_workflow_tags.yaml", "hello_workflow_invalid.yaml"])
131 def test_validate_workflow_tags_pass(fn):
132     # Test a fully valid config file passes schema
133     with resources.as_file(resources.files("uwtools.resources")) as resc:
134         schema_file = resc / "rocoto.jsonschema"
135 
136     config_file = support.fixture_path(fn)
137     assert validator.validate_yaml(
138         schema_file=schema_file, config_file=config_file, check_path=False
139     )
140 
141     assert not validator.validate_yaml(
142         schema_file=schema_file, config_file=config_file, check_path=False
143     )
```

@maddenp-noaa would you consider trying to do this? I'm not sure if this is possible if I want to do an `assert not` and an `assert` with two different files. Thoughts?

Reply:
I think I'd break them up or, more likely, `parametrize` them as you're doing. Since `validator.validate_yaml()` returns a `bool`, I think we could e.g.
```
@pytest.mark.parametrize("vals", [("hello_workflow_tags.yaml", True), ("hello_workflow_invalid.yaml", False)])
def test_validate_workflow_tags(vals):
    fn, validity = vals
    with resources.as_file(resources.files("uwtools.resources")) as resc:
        schema_file = resc / "rocoto.jsonschema"
    config_file = support.fixture_path(fn)
    assert validator.validate_yaml(
        schema_file=schema_file, config_file=config_file, check_path=False
    ) is validity
```
`is validity` will be either `is True` or `is False` depending on the `vals` parameter.